tests/examples/test_quantization_w8a8_fp8.py

```diff
@@ -0,0 +1,56 @@
+import shlex
+import shutil
+import sys
+from pathlib import Path
+
+import pytest
+
+from tests.examples.utils import (
+    ReadMe,
+    gen_cmd_fail_message,
+    requires_gpu,
+    requires_torch,
+)
+from tests.testing_utils import run_cli_command
+
+
+@pytest.fixture
+def example_dir() -> str:
+    return "examples/quantization_w8a8_fp8"
+
+
+@pytest.mark.example
+@requires_gpu
+@requires_torch
+class TestQuantizationW8A8_FP8:
+    """
+    Tests for examples in the "quantization_w8a8_fp8" example folder.
+    """
+
+    def test_doc_example_command(self, example_dir: str, tmp_path: Path):
```

Comment:
we seem to repeat the code for this test case across the different examples - is there anyway we can reuse some of the functionality?

Reply:
I’ll take a look through today. There is probably a balance to abstract some of it while not obfuscating what the test is doing to shave off a bit of the duplication.

Reply:
I’ve added a pair of helper functions that reduce duplication in most test implementation code.